fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ParamExceptionMapper.java

```diff
@@ -38,7 +38,7 @@
 
     @Override
     public Response toResponse(final ParamException e) {
-        LOGGER.debug("ParamException intercepted by ParamExceptionMapper: \n", e);
+        LOGGER.error("ParamException intercepted by ParamExceptionMapper: \n", e);
```

Comment:
Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`